### ISSUE TITLE
Errors with localized keys

### ISSUE THREAD

**Issue by davidskeck** [ryonakano/pinit#93]
Hello,

When I use Pin It! to create a .desktop file. I get the following errors if I then run `desktop-file-validate` on the resulting file.

```
.local/share/applications/com.logseq.desktop: error: key "Name[en_US.UTF-8]" in group "Desktop Entry" is a localized key, but there is no non-localized key "Name"
.local/share/applications/com.logseq.desktop: error: key "Comment[en_US.UTF-8]" in group "Desktop Entry" is a localized key, but there is no non-localized key "Comment"
.local/share/applications/com.logseq.desktop: error: required key "Name" in group "Desktop Entry" is not present
```

**Comment by ryonakano** [comment]
Confirmed, looks like we need a non-localized key per localized key

```
ryo@lb450m:~$ cat .local/share/applications/wawa.desktop
[Desktop Entry]
Name[ja]=awawa
Comment[ja]=awawa
Exec=sawa
Categories=Network;
Type=Application
Terminal=false
ryo@lb450m:~$ desktop-file-validate .local/share/applications/wawa.desktop 
.local/share/applications/wawa.desktop: error: key "Name[ja]" in group "Desktop Entry" is a localized key, but there is no non-localized key "Name"
.local/share/applications/wawa.desktop: error: key "Comment[ja]" in group "Desktop Entry" is a localized key, but there is no non-localized key "Comment"
.local/share/applications/wawa.desktop: error: required key "Name" in group "Desktop Entry" is not present
```

**Comment by ryonakano** [comment]
Should be fixed in #109. We no longer writes localized key
